frontend/simple/views/components/Tooltip.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <span @mouseenter="show" @mouseleave="hide">
+    <slot></slot>
+    <div
+      class="has-background-dark has-text-grey-lighter is-bottom is-size-7 c-tooltip"
+      :style="stylesPosition"
+      v-if="isActive"
+      v-append-to-body
+    >
+      <slot name="tooltip"></slot>
+    </div>
+  </span>
+</template>
+<style lang="scss" scoped>
+@import "../../assets/sass/theme/index";
+
+.c-wrapper {
+  display: inline-block;
+}
+
+.c-tooltip {
+  position: absolute;
+  top: 0;
+  left: 0;
+  border-radius: $radius;
+  padding: $gi-spacer-sm;
+  opacity: 0.95;
+  // TODO create SCSS variables with possible z-index (hover,modals, header, etc...)
+  // So we don't end up with 99999999 values
+  z-index: 50;
+  max-width: 12rem;
+}
+</style>
+<script>
+/*
+NOTE: when needed, this component can be improved with more options:
+- Click instead of hover
+- More directions
+- Controlled tooltip from outside
+For now I've just did the needed API for this particular task but I think it's pretty easy to scale it.
+*/
+export default {
+  name: 'Tooltip',
+  props: {
+    direction: {
+      type: String,
+      default: 'bottom'
+    }
+  },
+  data: () => ({
+    trigger: null,
```

Comment:
What is `trigger`? It doesn't seem to ever be used?

Reply:
Yes, you can see it being used at line 65, where it's used to get the component el properties:

```js
const { width, height, left, top } = this.trigger || this.$el.getBoundingClientRect()
```

Reply:
But it's always `null`, do it doesn't do anything :-\

Reply:
Ups, forgot a line in the middle. Fixed!